tests/integration.rs

```diff
@@ -503,3 +517,302 @@ async fn all_around_tests(client: Client) -> Result<(), Box<dyn std::error::Erro
 
     Ok(())
 }
+
+static PROJECTION_FILE: &'static str = include_str!("fixtures/projection.js");
+static PROJECTION_UPDATED_FILE: &'static str = include_str!("fixtures/projection-updated.js");
+
+async fn create_continuous_projection(
```

Comment:
**SUGGESTION:** I think that `continuous` is redundant here and in other places, as we'll have now only `continuous` projections.

Reply:
done